File: 27-remove-element/remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        n = len(nums)

        idx = 0

        for i in range(0, n):
            if (nums[i] != val):
                nums[idx] = nums[i]
                idx += 1
        
        return idx

File: 27-remove-element/remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int n = nums.length;

        int idx = 0;

        for (int i = 0; i < n; i++) {
            if (nums[i] != val) {
                nums[idx] = nums[i];
                idx++;
            }
        }

        return idx;
    }
}